scripts/Publications_data.R

```diff
@@ -0,0 +1,56 @@
+## Script to compile all publications currently associated with the IV dataset and catalog
+## which TRNs are found in which section of each publication
+
+##############################################################################################
+
+library(tidyverse)
+library(readr)
+library(here)
+library(fs)
+library(lubridate)
+library(stringr)
+
+dir_processed <- here("data", "processed")
+
+trials <- read_csv(path(dir_processed, "trials.csv"))
+cross_registrations <- read_rds(path(dir_processed, "trn", "cross-registrations.rds"))
+
+# 'trials' gives us more information about URL and pub_type to supplement what we know from 'cross_registration'
+pubs_supplementary_info <- trials %>%
+                           select(doi, url, publication_type) %>%
+                           unique()
```

Comment:
not all trials have a publication, and not all trials with a publication have a DOI and/or PubMed identifier (PMID) which are the two main persistent identifiers we have for publications. More publications have a DOI than a PMID. One option would be to filter for `has_publication` and then filter out all publications without a DOI, since you're later on DOI and need this (but note you'll be missing a few pubs with only a PMID). You could also filter for pubs with `publication_type == journal publication` although not necessary as the cross-registrations dataset only includes these types of pubs anyway.

I'm also questioning whether we really need to preserve the URLs here. The IntoValue dataset includes some duplicate trials with the same publication but different URLs (depending on whether the manual extractor recorded the URL from PubMed or the journal directly). Preserving these is causing complications with the join later (many-to-many). I would drop it unless we need it for something specific.